model/lifecycle.go

```diff
@@ -852,9 +852,8 @@ func createTasksForBuild(project *Project, buildVariant *BuildVariant, b *build.
 			}
 		}
 
-		// Display tasks depend on all exec task dependencies
+		// Local pointer to display task
 		if displayTask, ok := displayTasks[newTask.DisplayName]; ok {
-			displayTask.DependsOn = append(displayTask.DependsOn, newTask.DependsOn...)
```

Comment:
@ybrill was this something needed for EVG-5960?

Reply:
I was recently wondering the same thing 😄 
I remember it fixed something, but I don't remember what. Display tasks don't have dependencies and other tasks can't depend on them, so it seems strange to give them dependencies.

One thing that comes to mind is a display task is shown as blocked in the UI when any of its exec tasks are blocked. By adding dependencies to the display task we don't need to check the blocked status of each of the exec tasks on page load.

Reply:
Maybe to minimize risk I'll just change this to not have duplicates rather than removing dependencies entirely.